tests/features/dcatcommandline.feature

```diff
@@ -36,7 +36,7 @@ Feature: Testing the csvw command group in the CLI
           pmdcat:graph <http://data-graph-uri>;
           pmdcat:metadataGraph <http://catalog-metadata-graph-uri>;
           dct:creator <https://www.gov.uk/government/organisations/hm-revenue-customs>;
-          dct:description "All bulletins provide details on percentage of one litre or less bottles. This information is provided on a yearly basis."@en;
+          pmdcat:markdownDescription "All bulletins provide details on percentage of one litre or less bottles. This information is provided on a yearly basis."^^<https://www.w3.org/ns/iana/media-types/text/markdown#Resource>;
```

Comment:
Do we not want to say the Markdown is in English?

Reply:
it i sn't possible to set the lang and datatype at the same time.
 ```
>>> g.add((URIRef("http://subj"), URIRef("http://predicate"), Literal("hello world", "en", datatype="http://some-datatype")))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/csvcubed-pmd-venv/lib/python3.10/site-packages/rdflib/term.py", line 553, in __new__
    raise TypeError(
TypeError: A Literal can only have one of lang or datatype, per http://www.w3.org/TR/rdf-concepts/#section-Graph-Literal
```